src/commands/starter/starter.command.ts

```diff
@@ -0,0 +1,121 @@
+import { Message, RichEmbed, User } from 'discord.js';
+import { prefixedCommandRuleTemplate , defaultEmbedColor } from '../../config';
+import {
+    Command,
+    CommandClass,
+    detectStaff
+  } from '../../shared';
+
+@Command({
+    matches: ['starter', 'starterpack', 'starterkit'],
+    ...prefixedCommandRuleTemplate
+})
+
+    /**
+     * @param msg
```

Comment:
So this is the JsDoc for this function. Right now, it's not offering any information that isn't already there (it's just naming the parameters). Since this function is relatively simple, all things considered, let's delete this JsDoc